firmware/console/binary/tunerstudio.cpp

```diff
@@ -181,7 +181,7 @@ static uint8_t* getWorkingPageAddr(TsChannelBase* tsChannel, size_t page, size_t
 #endif
 	default:
 // technical dept: TS seems to try to read the 3 pages sequentially, does not look like we properly handle 'EFI_TS_SCATTER=FALSE'
-		criticalError("unhandled getWorkingPageAddr");
+		tunerStudioError(tsChannel, "ERROR: page address out of range");
 		return nullptr;
```

Comment:
'return ptr' i have a feeling that it would cause a seg fault? look at getWorkingPageAddr() usages?

Reply:
> Could it be that we have outdated code in the library used by wideband, and that's where the confusion comes from?

ooops, that's what got me confused